rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Each Team or Working Group is able to set its own pace and objectives quarterly.
+
+## Product/Service Teams and Working Groups
+
+### Teams (Product Teams and Service Teams)
+
+Some Teams focus on producing and maintaining _products_ (ie. go-ipfs, ipfs-cluster, etc.) while other Teams focus on providing _services_ for their audiences to rely on (ie. Infrastructure). The rules and guidelines here apply equally to both Product Teams and Services Teams,
+
+- A Team must have at least 3 people whose Full Time efforts are primarily focused on that Team's work.
+- Teams define  [Objectives and Key Results](https://weekdone.com/resources/objectives-key-results) (OKRs) every quarter. Members of a Team primarily derive their tasks from those OKRs.
+- Some Teams have multiple products or services under their view. Others are focused on a single product or service. Teams can also _add_ products and services to their portfolio at any time.
+- **Teams should subdivide when the Team gets bigger than 3-5 Full Time people** so that the group has a manageable amount of OKRs. When a  Team becomes big enough to subdivide and specialize, they can split into sub-teams. In that case, those sub-teams can create their own OKRs but they must coordinate with each other to create shared, overarching OKRs and Roadmaps. This allows external audiences to follow the work of the overall Team without having to read every sub-team's OKRs.
+- People _can_ belong to multiple Teams, but should avoid over committing. **People whose time is split across teams do not count toward either Team's 3-person minimum.**
+- Captains (team leads) are responsible for keeping track of whether people are over committed (including keeping track of whether people have overcommitted across multiple Teams)
+
+### Working Groups
+
+- A Working Group can't form until there are multiple Teams that need it to exist
+- Working Groups don't necessarily have any people allocating full-time effort to that group's goals. As a result, Working Groups are likely to push work into the OKRs of Teams, which can more easily allocate people's time.
+- Working Groups occasionally define OKRs, but they don't have to define OKRs every quarter.
+- Working Groups don’t necessarily subdivide when they get bigger. Unlike Teams, which are encouraged to form a tree of sub-teams when they grow bigger than ~5 people, Working Groups are able to convene larger groups of stakeholders.
+
+### The Core Working Group
```

Comment:
I suggest to change this to something else than `Core`, as `Core` has different meaning in Protocol Labs and this may lead to confusion.

`Coordination working group`, `Steering commitee/WG` etc. would be clearer

Reply:
+1 (this was confusing for me as well)